src/lib/variants.ts

```diff
@@ -3,13 +3,14 @@ import {
     ViewLog,
     UserCohort,
 } from '../components/modules/epics/ContributionsEpicTypes';
-import { shouldThrottle, shouldNotRenderEpic, userIsInTest } from '../lib/targeting';
-import { getCountryName, inCountryGroups, CountryGroupId } from '../lib/geolocation';
-import { getArticleViewCountForWeeks, historyWithinArticlesViewedSettings } from '../lib/history';
-import { isRecentOneOffContributor } from '../lib/dates';
```

Comment:
haha good spot!

Reply:
I can't take credit for that one, IntelliJ pointed it out!